core/net6.0/proxy/Apache.OpenWhisk.Runtime.Common/Init.cs

```diff
@@ -54,58 +53,52 @@ public async Task<Run> HandleRequest(HttpContext httpContext)
                 {
                     await httpContext.Response.WriteError("Cannot initialize the action more than once.");
                     Console.Error.WriteLine("Cannot initialize the action more than once.");
-                    return (new Run(Type, Method, Constructor, AwaitableMethod));
+                    return new Run(_type, _method, _constructor, _awaitableMethod);
                 }
 
-                string body = await new StreamReader(httpContext.Request.Body).ReadToEndAsync();
+                using StreamReader reader = new(httpContext.Request.Body);
+                string body = await reader.ReadToEndAsync();
                 JObject inputObject = JObject.Parse(body);
-                if (!inputObject.ContainsKey("value"))
+                if (!inputObject.TryGetValue("value", out JToken? message) || message is not JObject valueObj)
                 {
                     await httpContext.Response.WriteError("Missing main/no code to execute.");
-                    return (null);
+                    return null;
```

Comment:
I am not sure why all return values are between parentheses. Is this enforced by the Scalariform plugin?